.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: pop release
+
+on:
+  release:
+    types: [ published ]
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: ref to build binary from
+        required: false
+  push:
+    branches:
+      - 'frank/build-release'
```

Comment:
Presumably, this is just here for initial testing?

Reply:
Yeah, it's the only way to have the workflow appear in GH UI pre-merge to main.

Reply:
Resolved in latest commit